woocommerce/class-sv-wc-helper.php

```diff
@@ -979,6 +982,24 @@ public static function is_enhanced_admin_screen() {
 	}
 
 
+	/**
+	 * Determines whether the new WooCommerce enhanced navigation is supported and enabled.
+	 *
+	 * @since 5.10.6
+	 *
+	 * @return bool
+	 */
+	public static function is_wc_navigation_enabled() {
```

Comment:
The reason I didn't use "enhanced" in this method name is because the word "enhanced" is not used by WooCommerce in the Features section nor in its namespaces.

I'm ok with using "enhanced" for comments or even locally in each plugin.

However, in the future, if WooCommerce decides to create an enhanced navigation as a plugin or another feature that is not enabled by default, for example, we'd have to rename this method and in every plugin calling it. So, I opted to a more generic name. Does that make sense? If you feel that we should use `is_enhanced_admin_screen` either way, it's also ok.

Reply:
sure, that's why I didn't feel strong about it - I believe we adopted the term enhanced ourselves to make "wc_admin" less ambiguous and in this helper class there's a bit of everything - I think you can leave as since we started implementing it, saves us a few updates